src/custom/components/AccountDetails/Transaction.tsx

```diff
@@ -307,15 +309,19 @@ function ActivitySummary(params: {
   const isOrder = type === ActivityType.ORDER
 
   if (isOrder) {
-    const { inputToken, sellAmount, feeAmount, outputToken, buyAmount, validTo, kind } = activity as Order
+    const { inputToken, sellAmount, feeAmount, outputToken, buyAmount, validTo, kind, fulfillmentTime } =
+      activity as Order
 
     const sellAmt = CurrencyAmount.fromRawAmount(inputToken, sellAmount.toString())
     const feeAmt = CurrencyAmount.fromRawAmount(inputToken, feeAmount.toString())
     const outputAmount = CurrencyAmount.fromRawAmount(outputToken, buyAmount.toString())
+    const limitPrice = formatSmart(getLimitPrice(activity as Order))
 
     orderSummary.from = `${formatSmart(sellAmt.add(feeAmt))} ${sellAmt.currency.symbol}`
     orderSummary.to = `${formatSmart(outputAmount)} ${outputAmount.currency.symbol}`
+    orderSummary.limitPrice = `${limitPrice} ${outputAmount.currency.symbol} per ${sellAmt.currency.symbol} `
```

Comment:
Better to get the `sellAmt.currency.symbol` and `outputAmount.currency.symbol` from the `limitPrice` return value? (`baseCurrency` & `quoteCurrency` from Price).

Reply:
You could do that, but I don't see the need here.